test/navigation/nonsipp/SharesDisposalNavigatorSpec.scala

```diff
@@ -206,12 +206,28 @@ class SharesDisposalNavigatorSpec extends BaseSpec with NavigatorBehaviours {
               disposalIndex,
               HowManySharesRedeemedPage,
               (srn, shareIndex: Max5000, disposalIndex: Max50, _) =>
-                controllers.routes.UnauthorisedController.onPageLoad()
+                controllers.nonsipp.sharesdisposal.routes.TotalConsiderationSharesRedeemedController
+                  .onPageLoad(srn, shareIndex, disposalIndex, NormalMode)
             )
             .withName("go from How Many Shares Redeemed page to Unauthorised page")
```

Comment:
This shouldn't stop the PR from getting approved, but just as a reminder, that the test descriptions should be updated to show the name of the destination page once that page has been implemented. This can be done in a future ticket, no need to change it here.